main.rb

```diff
@@ -0,0 +1,131 @@
+require './app'
+
+def welcome
+  puts 'Welcome to the School Library App'
+end
+
+def create_person(library)
+  print 'Do you want to create a Student (1) or a teacher (2)? [Input the number]:'
+
+  input = gets.chomp.to_i
+
+  print 'Name: '
+  name = gets.chomp
+
+  print 'Age: '
+  age = gets.chomp.to_i
+
+  if input == 1
+    print 'Has parent permission? y/n :'
+    permission = gets.chomp
+
+    library.create_student(age, name, permission == 'y')
+  else
+    print 'Specialization:'
+    spec = gets.chomp
+
+    library.create_teacher(age, name, spec)
+  end
+
+  puts input == 1 ? 'Student created Successfuly!' : 'Teacher created Successfuly!'
+
+  user_choice(library)
+end
```

Comment:
- [ ] Great job so far, @francis3400! However, as the previous reviewer mentioned, I'm also unable to create a person in your project. The error message I receive is displayed below. I kindly request that you fix this issue so that both students and teachers can be created successfully. :100: 

- [ ] Additionally, once you've fixed the above issue, please make sure that rentals can also be created successfully. This will allow the next reviewer to test both functionalities :wink: 

- Error Message
![image](https://user-images.githubusercontent.com/102699958/233789814-3dab67f2-6817-429c-9f15-7687960f7343.png)